src/NuGet.Clients/NuGet.CommandLine/NuGetCommand.resx

```diff
@@ -5245,4 +5245,18 @@ nuget update -Self</value>
   <data name="CommandMSBuildVersion" xml:space="preserve">
     <value>Specifies the version of MSBuild to be used with this command. Supported values are 4, 12, 14. By default the MSBuild in your path is picked, otherwise it defaults to the highest installed version of MSBuild.</value>
   </data>
+  <data name="ClearCacheCommandDescription" xml:space="preserve">
+    <value>Clears your local NuGet cache.</value>
```

Comment:
Tried to improve the description in https://github.com/NuGet/NuGet.Client/commit/ef6950899c19afaa3460a4e3e7faef4b2b54e6f9